### PR TITLE
Feature/ue 5.3

### PR DESCRIPTION
**Pull request by YasserNezzari**

### CODE REVIEW

File: Source/UltraleapTrackingCore/Private/LeapComponent.cpp [hunk 1]
```diff
@@ -388,8 +388,6 @@ void ULeapComponent::OnDeviceAdded(IHandTrackingWrapper* DeviceWrapper)
 	{
 		RefreshDeviceList(true);
 	}
-	ActiveDeviceSerial = DeviceWrapper->GetDeviceSerial();
```
Comment (MaxPalmer-UH) [inline Source/UltraleapTrackingCore/Private/LeapComponent.cpp:391]: Why was this line deleted?

**Review by MaxPalmer-UH — COMMENTED** [review]
Can you review the comment please?

File: Source/UltraleapTrackingCore/Private/LeapComponent.cpp (hunk 1 above)
Comment (YasserNezzari) [reply Source/UltraleapTrackingCore/Private/LeapComponent.cpp:391]: it's a fix to pinch and grab when openXR is not available: https://ultrahaptics.atlassian.net/browse/UNREAL-729

File: Source/UltraleapTrackingCore/Private/FUltraleapDevice.cpp [hunk 2]
```diff
@@ -1320,98 +1320,14 @@ void FUltraleapDevice::SetOptions(const FLeapOptions& InOptions)
...
-					break;
-				default:
-					break;
-			}
-
-			// let's use basic vive settings for cardboard for now
-		}
-		// Other, e.g. cardboard
-		else
+		else	// For android
```
[the remark this answers is not in the document]
Comment (YasserNezzari) [reply Source/UltraleapTrackingCore/Private/FUltraleapDevice.cpp:1323]: Do you suggest keeping some of the other exceptions?

[the remark this answers is not in the document]
Comment (MaxPalmer-UH) [reply Source/UltraleapTrackingCore/Private/FUltraleapDevice.cpp:1323]: No, for the moment let's leave it as is.

File: Source/UltraleapTrackingCore/Private/LeapComponent.cpp (hunk 1 above)
Comment (MaxPalmer-UH) [reply Source/UltraleapTrackingCore/Private/LeapComponent.cpp:391]: Ok, thanks.
